clients/extension/popup.html

```diff
@@ -15,51 +15,40 @@
   <link rel="stylesheet" href="styles/lighthouse-loading.css">
 </head>
 <body>
-
-  <header class="header">
-    <img src="images/lh_logo_bg.png" class="header__icon" width="100" height="100" alt="Lighthouse Logo">
-    <div class="header-titles">
-      <h1 class="header-titles__main">Lighthouse</h1>
-      <h2 class="header-titles__url">...</h2>
-      <div class="header-titles__throttling">
-        <label for="lantern-checkbox">
-          <input
-            id="lantern-checkbox"
-            type="checkbox"
-            checked />
-          Simulated throttling (faster)
-        </label>
-      </div>
-    </div>
-  </header>
-
   <main class="main" role="main">
-    <div class="feedback"></div>
-
-    <button class="button button--configure" id="configure-options">Options</button>
-    <button class="button button--generate" id="generate-report">Generate report</button>
-  </main>
-
-  <!-- Show running view initially (.subpage--visible). -->
-  <aside class="status subpage subpage--visible">
-    <div class="lighthouse-effects">
-      <img src="images/lh_logo_bg_no-light.png" alt="LH logo">
-      <div class="lighthouse-effects__wrapper">
-        <div class="lighthouse-effects__light"></div>
+    <div class="section section--run">
+      <div class="lighthouse-effects">
+        <img src="images/lh_logo_bg_no-light.png" class="header__icon" width="100" height="100" alt="Lighthouse Logo">
+        <div class="lighthouse-effects__wrapper animate">
+          <div class="lighthouse-effects__light"></div>
+        </div>
+      </div>
+      <div class="button--generate__wrapper">
+        <button class="button button--generate" id="generate-report">Generate report</button>
+        <button class="button button--configure" id="configure-options">Options</button>
+        <span>Uses the <a href="https://developers.google.com/speed/pagespeed/insights/" target="_blank" rel="nofollow">PSI</a> API</span>
       </div>
     </div>
-    <div class="status__msg">Starting...</div>
-    <div><small class="status__detailsmsg"></small></div>
-  </aside>
+    <div class="section section--devtools-info">
+      You can also run Lighthouse locally via the DevTools Audits panel.
+      Shortcut to open DevTools: Cmd + Shift + I or F12
```

Comment:
we'll need to handle Ctrl as well..

here's how devtools UI shows cmd/ctrl conditionaly: https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/ui/KeyboardShortcut.js?l=180-181&rcl=7ea553ab9cfc2f79ad374abcd4f8960966248239